UpdateService/handler.ts

```diff
@@ -161,7 +161,7 @@ export function UpdateServiceHandler(
       telemetryClient.trackEvent({
         name: "api.services.update",
         properties: {
-          isVisible: isVisible ? "false" : String(isVisible),
+          isVisible: isVisible ? String(isVisible) : "false",
```

Comment:
isn't this enough
```suggestion
          isVisible: String(isVisible)
```
or even

```suggestion
          isVisible
```

Reply:
Fixed in [821fa9e](https://github.com/pagopa/io-functions-services/pull/87/commits/821fa9e7bdef60a9b32e956dac12af7d1eacd21a)